inc/libiptnt.h

```diff
@@ -88,8 +88,8 @@ typedef struct _IPT_TRACE_DATA
 
 typedef struct _IPT_TRACE_HEADER
 {
-    DWORD ThreadId;
-    DWORD Unknown;
+    ULONG ThreadId;
```

Comment:
OK, I verified with x86 -- there's no other field, this should just be ULONG64 in NT, and DWORD64 in Win32. Please make the change (and necessary casts where needed).